### PR TITLE
[application] James W

### PR DESCRIPTION
**Pull request by jsdw**

### CODE REVIEW

File: join_request/jameswilson.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+collective: Potoc
+member:
+  name: James Wilson
+  address: 12TNvHiRkwzYqT5UZ86cfUvBeZBjLLYUzHLa4Hix99oTrgqT
+  github: jsdw
```
Comment (wirednkod) [inline join_request/jameswilson.yml:5]: please fill your matrix handle as per https://github.com/super-collective/collective-cli/pull/4#discussion_r1586380064
```suggestion
  address: 12TNvHiRkwzYqT5UZ86cfUvBeZBjLLYUzHLa4Hix99oTrgqT
  matrix:
  github: jsdw
```

Comment (jsdw) [inline join_request/jameswilson.yml:5]:
```suggestion
  github: jsdw
  matrix: @james.wilson:parity.io
```

Comment (jsdw) [reply join_request/jameswilson.yml:5]: Done!

**Review by ggwpez — APPROVED** [review]
Accepted for work on relevant projects (subxt, scale-* libs).

**Pull request merged**